### ISSUE TITLE
Data.gov meets Identifier component of USWDS

### ISSUE THREAD

**Issue by hkdctol** [GSA/data.gov#3098]
## User Story

In order to meet USWDS, the new Data.gov should meet the Identifier component of USWDS

## Acceptance Criteria
- [ ] GIVEN a user navigates to Data.gov \
  THEN user sees the identifying information about Data.gov's parent agency, and other information from the Identifier Component of USWDS.
  
## Background
See https://designsystem.digital.gov/components/identifier/
Along with the Banner component, the Identifier component provides useful information to the public about parent agency and provides links to all required sites

## Sketch
- [ ] Review guidance on https://designsystem.digital.gov/components/identifier/
- [ ] Add the Default information (GSA as parent agency) and other required links, to Data.gov, see https://digital.gov/resources/required-web-content-and-links/?dg for the required links to put in footer, following the guidance at: https://designsystem.digital.gov/components/footer/

**Comment by hkdctol** [comment]
Sites that need this: resources, catalog, data.gov, sdg.data.gov, and dashboard. See the tracking spreadsheet at https://docs.google.com/spreadsheets/d/14GUVegR3Gy2jWduSyEpuwirKpwctTyvk23oY9HoYQi0/edit?usp=sharing

**Comment by hkdctol** [comment]
Work on #3891 will accomplish this for Data.gov main site, but still need for resources.data.gov, strategy.data.gov, and sdg.data.gov

**Comment by hkdctol** [comment]
Specifically, the scanning in late August will check for required links on Accessibility, FOIA, and Privacy.

**Comment by hkdctol** [comment]
Need the same footer/links on catalog.data.gov, but that won't be checked in August. We can create a separate ticket later if needed.

**Comment by btylerburton** [comment]
Blocked by catalog pipeline issues.
